### PR TITLE
Exclude `binder`, warn in `MANIFEST.in`, ignore `.ipynb_checkpoints`

### PR DESCRIPTION
**Pull request by krassowski**
Addresses #237 and excludes `.ipynb_checkpoints` from coverage to avoid spam warnings when running jest

### CODE REVIEW

File: {{cookiecutter.python_name}}/MANIFEST.in [hunk 1]
```diff
@@ -1,3 +1,4 @@
+# WARNING: MANIFEST.in has no effect for hatch backend
```
Comment (fcollonval) [inline {{cookiecutter.python_name}}/MANIFEST.in]: Should we delete this file as the cookiecutter is also for beginner and this may add more confusion?

**Review by fcollonval — COMMENTED** [review]
Thanks @krassowski

**Review by fcollonval — APPROVED** [review]
Thanks @krassowski 

> CI failure are not related

**Comment by krassowski** [comment]
@meeseeksdev please backport to 3.0

**Pull request merged**